app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -161,6 +163,41 @@ def unlink_from_work
           work.rendering_ids -= [file_set.id]
           work.save!
         end
+
+        # method to use when fully valkyrized
```

Comment:
if this is a public method, i think it needs more descriptive documentation.

Reply:
It is a private method.  That description probably should be a TODO.

Reply:
i think my concern is about the phrase "fully valkyrized". this doesn't help me understand when i should use this method, or when i should look to remove the others.

Reply:
I had switched the comment to...
```
# TODO: Use this method where ever perform_save is called when fully valkyrized
```

but if your hesitation is 'fully valkyrized' (which is used in many comments), then perhaps a more accurate description would be...

```
# TODO: Use this method where ever perform_save is called when env passes resources instead of active fedora objects.
```

Reply:
Comment updated to be more precise...

`# TODO: Change calls to `#perform_save` to call `#save` instead when env passes resources instead of active fedora objects`